components/suggestion/suggestion_box.jsx

```diff
@@ -268,10 +268,7 @@ export default class SuggestionBox extends React.Component {
         const textbox = this.getTextbox();
         const pretext = textbox.value.substring(0, textbox.selectionEnd);
 
-        if (!this.composing &&
-            SuggestionStore.getPretext(this.suggestionId) !== pretext &&
-            (this.props.openWhenEmpty || pretext.length >= this.props.requiredCharacters)
```

Comment:
It seems like we don't need this condition at all, it's rarely hit and didn't cause any issues when removed (and fixed the bug).